### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by femithz**
This branch is created for testing purpose
We have the test folder 
Inside the test folder we have the auth.js file
the auth.js file is mainly to created to test the auth section of the app.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+ - "stable"
+services:
+ - mongodb
+ script: npm test
```
Comment (raphaelakpan) [inline .travis.yml]: Always put a new line at the end of every file, otherwise you will see this red arrow. The new line helps git.

**Comment by raphaelakpan** [comment]
Looks good. We can merge this so that you can pull in the changes and complete the work for creating and authenticating users.

In your other PR, when you see comments from Hound, they indicate style violations and you should address then for example missing semicolon.

For the message
 > 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

You can check this post on how to silence the warning https://stackoverflow.com/questions/27441803/why-does-jshint-throw-a-warning-if-i-am-using-const

**Comment by raphaelakpan** [comment]
I also had a comment here that you didn't address. Didn't you understand it? https://github.com/femithz/birthday_reminder/pull/3#discussion_r305454178

**Comment by raphaelakpan** [comment]
Try this option of creating a `.jshintrc` file
image

File: .travis.yml (hunk 1 above)
Comment (femithz) [reply .travis.yml]: okay sir,that done in the new pull request sir.

**Pull request merged**